lib/tasks/certs.rake

```diff
@@ -117,4 +117,44 @@ namespace :certs do
     puts ''
     puts 'Double check https://fpki.idmanagement.gov/notifications/#notifications for new and revoked certs'
   end
+
+  # This task uses existing certificates' CA Repository information from the subject information
+  # access extension to see if existing certs have issued any certificates that we do not have.
+  # The find_replacement task is helpful for expiring certificates, but this task is useful
+  # when a certificate authority has issued new certificates that are not replacements.
+  # Currently, the task only fetches certs issued by our trusted root certificates, but could be
+  # modified to go deeper in the issuing tree.
+  desc 'Find certs issued by existing certs'
+  task find_new: :environment do
+    root_keys = IdentityConfig.store.trusted_ca_root_identifiers + IdentityConfig.store.dod_root_identifiers
+    root_keys.each do |key_id|
+      root_cert = CertificateStore.instance[key_id]
+      repository_certs = IssuingCaService.fetch_ca_repository_certs_for_cert(root_cert)
+
+      repository_certs.each do |repo_cert|
+        next if CertificateStore.instance.certificates.any? { |x| x.serial == repo_cert.serial }
```

Comment:
Do we need to load the certificate store after each cert is written?

Reply:
We _should_ be able to trust PKI stuff to not collide, but added in 084a09e for safety